docs/user-guide/devguide.md

```diff
@@ -300,6 +300,72 @@ The Arduino example provided above will print the message on the console. Keep i
 
 Refer to the [Observability](observability.md) guide if you want to learn about how to use the starter kit observability features.
 
+### LNS discovery
```

Comment:
nit: we could also move this to a separate doc

Reply:
Done with 5dce35ec6ddb9203665717b0bdfaa75760985b9a